modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -194,6 +194,14 @@ As a consequence, since the selection is still on the first element, the `List P
 
 As a result, the selection is in the correct place and the `List Properties...` context menu item is `enabled`.
 
+=== `TableOperations.opEraseRows` did not calculate the correct row index for colgroup tables.
```

Comment:
`TableOperations.opEraseRows` is a non-public API, so should this reference the command `mceTableDeleteRow` maybe?

Reply:
I feel like `mceTableDeleteRow` would be the best option.

Reply:
```suggestion
=== `mceTableDeleteRow` did not calculate the correct row index for colgroup tables.
```